src/renderer/components/proxy-settings/proxy-settings.css

```diff
@@ -1,3 +1,15 @@
 .protocol-dropdown {
   margin-block-end: 1rem;
 }
+
+.proxy-warning {
+  font-weight: bold;
+  padding-inline: 4%;
+  background-color: var(--accent-color);
+  color: var(--text-with-accent-color);
+  padding-block: 3px;
```

Comment:
```suggestion
  padding-block: 12px;
  padding-inline: 4%;
  background-color: var(--accent-color);
  color: var(--text-with-accent-color);
  font-weight: bold;
```
![image](https://github.com/user-attachments/assets/57ebff4b-0e22-4667-b9ee-e5528fef8fa4)